pii_recognizer/function.yaml

```diff
@@ -0,0 +1,120 @@
+    base_image: mlrun/mlrun
+    commands: []
+    code_origin: git@github.com-personal:pengwei715/functions.git#64a654de189cb728fe97a7e534fd391c22cd9c4e:pii_recognizer.py
+    origin_filename: pii_recognizer.py
+    requirements:
+    - mlrun==1.4.0-rc19
+    - pandas
+    - presidio-anonymizer
+    - presidio-analyzer
+    - torch
+    - flair@git+https://github.com/flairNLP/flair.git@d4ed67bf663e4066517f00397412510d90043653
+    - st-annotated-text
+    - https://huggingface.co/beki/en_spacy_pii_distilbert/resolve/main/en_spacy_pii_distilbert-any-py3-none-any.whl
+  entry_points:
+    analyze:
```

Comment:
Hi @guy1992l , I am not sure how to deal with this situation, This is a method inside of the FlairRecognizer class. it's used inside of the presidio_analyzer.AnalyzerEngine class. I am not sure why it's picked up. Can I just remove it manually in the function.yaml file?